base/dcp_sharded.go

```diff
@@ -328,7 +339,25 @@ func initCBGTManager(bucket Bucket, spec BucketSpec, cfgSG cbgt.Cfg, dbUUID stri
 
 	if spec.Auth != nil && spec.Certpath == "" {
 		username, password, _ := spec.Auth.GetCredentials()
-		addCbgtCredentials(dbName, username, password)
+		addCbgtCredentials(dbName, bucket.GetName(), username, password)
+	}
+
+	if spec.IsTLS() {
+		bucketUUID, err := bucket.UUID()
+		if err != nil {
+			return nil, fmt.Errorf("failed to fetch UUID of bucket %v: %w", MD(bucket.GetName()).Redact(), err)
+		}
+		if spec.CACertPath != "" {
+			certs, err := loadCertificatesFromFile(spec.CACertPath)
+			if err != nil {
+				return nil, err
+			}
+			setCbgtRootCertsForBucket(bucketUUID, certs)
+		} else if spec.TLSSkipVerify {
+			setCbgtRootCertsForBucket(bucketUUID, nil)
+		} else {
+			setCbgtRootCertsForBucket(bucketUUID, getSystemCertPool())
+		}
```

Comment:
Have you already seen `getRootCAs()` ? It looks like it's mostly doing the same thing as these load/system cert pool utils.